Citizens/templates.yml terminatorgenisys

```diff
@@ -1246,6 +1246,38 @@ ironmanmkxv:
     uuid: 9b3e4ff2-136d-481c-8aa7-f858db93265b
     traitnames: spawned,type,inventory,owner,gravity,location
     name: sneaky
+terminatorgenisys:
```

Comment:
https://video.search.yahoo.com/yhs/search?fr=yhs-omr-001&hsimp=yhs-001&hspart=omr&p=the+terminator+beating+up+bill+paxton#id=1&vid=bda2c7df659042b251ae8ef4a2445911&action=click